Article/Article.js

```diff
@@ -1,20 +1,27 @@
 // Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
-
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.domElement = domElement;
+    // create a reference to the ".expandButton" class.
+    this.expandButton = this.domElement.querySelector(".expandButton");
```

Comment:
Just FYI, you could also write this as:
`this.expandButton = this.domElement.getElementsByClassName(expandButton)[0];`